modules/ROOT/pages/kafka/kafka-connector-reference.adoc

```diff
@@ -5,252 +5,639 @@ endif::[]
+| Maximum Polling Interval Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Determines the time unit for request timeout scalar. This parameter can be overridden at source level. |  `SECONDS` | 
+| Isolation Level a| Enumeration, one of:
+
+** READ_UNCOMMITTED
+** READ_COMMITTED a| Controls how to read messages written transactionally. 
+
+If set to `read_committed`, `consumer.poll()` will return only transactional messages that have been committed. If set to `read_uncommitted`' (default), `consumer.poll()` returns all messages, even transactional messages that were aborted. Non-transactional messages are returned unconditionally in either mode. Messages are always returned in offset order. Hence, in `read_committed` mode, `consumer.poll()` returns only messages up to the last stable offset (LSO), which is the one less than the offset of the first open transaction.
+
+In particular, any messages appearing after messages belonging to ongoing transactions are withheld until the relevant transaction is completed. As a result, `read_committed` consumers are not able to read up to the high watermark when there are in-flight transactions. Further, when in `read_committed` the `seekToEnd` method returns the LSO. |  `READ_UNCOMMITTED` | 
+| Exclude internal topics a| Boolean |  Whether internal topics matching a subscribed pattern should be excluded from the subscription. It is always possible to explicitly subscribe to an internal topic. |  `true` | 
+| Auto offset reset a| Enumeration, one of:
+
+** EARLIEST
+** LATEST
+** ERROR a| Determines what to do when there is no initial offset in Kafka or if the current offset no longer exists on the server (for example, because the data was deleted):
+
+* EARLIEST: Automatically reset the offset to the earliest offset. 
+* LATEST: Automatically reset the offset to the latest offset. 
+* ERROR: Throw an error if no previous offset is found for the consumer's group. | `LATEST` | 
+| Retry Backoff Timeout a| Number |  The amount of time to wait before attempting to retry a failed request to a given topic partition. This avoids repeatedly sending requests in a tight loop under some failure scenarios. |  `100` | 
+| Retry Backoff Timeout Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Determines the time unit for the reconnect backoff timeout scalar. |  `MILLISECONDS` | 
+| Check CRC a| Boolean |  Automatically check the CRC32 of the records consumed. This ensures that no on-the-wire or on-disk corruption to the messages occurred. This check adds some overhead, so in cases seeking extreme performance, this can be disabled. |  `true` | 
```

Comment:
Reword "in cases seeking extreme performance..." Maybe "in situations that require extremely high performance..."